app/package.json

```diff
@@ -23,10 +23,14 @@
     "test-chrome-headless": "yarn test && node node_modules/.bin/mocha-headless-chrome -f tests/index.html"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
+    "@ledgerhq/hw-transport-u2f": "^4.7.3",
+    "base-x": "^3.0.5",
```

Comment:
this isn't needed - even if you needed it, I recommed using the "base58" library exported by cardano-crypto.js https://github.com/vacuumlabs/cardano-crypto.js/blob/master/index.js#L692 to avoid adding unnecessary dependencies.